internal/util/testutil/pg.go

```diff
@@ -53,20 +53,28 @@ func PostgreSQLURL(tb testing.TB, opts *PostgreSQLURLOpts) string {
 		username = "readonly"
 	}
 
-	q := url.Values{
-		"pool_min_conns": []string{"1"},
+	u := &url.URL{
+		Scheme: "postgres",
+		User:   url.UserPassword(username, ""),
+		Host:   "127.0.0.1:5432",
+		Path:   databaseName,
 	}
+
+	AddTestParams(u)
+
+	q := u.Query()
 	for k, v := range opts.Params {
 		q.Set(k, v)
 	}
 
-	u := &url.URL{
-		Scheme:   "postgres",
-		User:     url.UserPassword(username, ""),
-		Host:     "127.0.0.1:5432",
-		Path:     databaseName,
-		RawQuery: q.Encode(),
-	}
+	u.RawQuery = q.Encode()
 
 	return u.String()
 }
+
+// AddTestParams adds testing parameters to the query.
+func AddTestParams(uri *url.URL) {
+	q := uri.Query()
+	q.Set("pool_min_conns", "1")
```

Comment:
Let's add to the `PostgreSQLURLF` itself and all `--postgresql-url` non-default values, and remove this function